src/Components/IndexScene/IndexScene.tsx

```diff
@@ -244,6 +245,7 @@ function getVariableSet(initialDS?: string, initialFilters?: AdHocVariableFilter
         hide: VariableHide.hideVariable,
       }),
       new CustomVariable({ name: VAR_LINE_FILTER, value: '', hide: VariableHide.hideVariable }),
+      new CustomVariable({ name: VAR_LOGS_FORMAT, value: '', hide: VariableHide.hideVariable }),
```

Comment:
Should we then remove ` new SceneVariableSet({ variables: [new CustomVariable({ name: VAR_LOGS_FORMAT, value: '' })] }),` from `ServiceScene`?

Reply:
this https://github.com/grafana/explore-logs/pull/330/files#diff-bf4a86d7b83030323394834b70dc2c1e8c8118278232286ed656b92dfca0e6fcL88-R87 ? yes 😆